week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -26,7 +24,7 @@ function mood() {
 
 function greaterThan10() {
   const num = 10;
-  const isBigEnough;
+  const isBigEnough = true;
```

Comment:
here you need to use a comparison operator. compare a num to a number and the function will return the true statement.